config/canonical.json

```diff
@@ -14245,17 +14249,19 @@
       "brand": "Chili's",
       "brand:wikidata": "Q1072948",
       "brand:wikipedia": "en:Chili's",
+      "cuisine": "tex-mex",
```

Comment:
#2250 adds another `tex-mex`. We should probably review all the other `mexican` entries to see whether any more should be retagged.

Reply:
It might be worth doing. Just doing a basic glance over it looks like there's some entries that could definitely be re-tagged.